app/scripts/lib/setupSentry.js

```diff
@@ -573,7 +594,7 @@ export default function setupSentry({ release, getState }) {
   const startSession = async () => {
     const hub = Sentry.getCurrentHub?.();
     const options = hub.getClient?.().getOptions?.() ?? {};
-    if (hub && (await getMetaMetricsEnabled()) === true) {
```

Comment:
I think you'll need to update most other uses of `getMetaMetricsEnabled` as well.